Index: app/src/main/java/com/example/myapplication/view/MainActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/example/myapplication/view/MainActivity.java	(date 1572564215000)
+++ app/src/main/java/com/example/myapplication/view/MainActivity.java	(date 1572564405645)
@@ -88,6 +88,7 @@
              }else{
                  Toast.makeText(getApplicationContext(),task.getException().getMessage(),
                          Toast.LENGTH_SHORT).show();
+
              }
 
          }
